### PR TITLE
[onert] Generate trainable sequence

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit generates trainable sequence for training backend.

ONE-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

**Review by ragmani — APPROVED** [review]
LGTM

File: runtime/onert/backend/train/BackendContext.cc [hunk 1]
```diff
@@ -74,12 +74,11 @@ FunctionMap BackendContext::genKernels()
 {
   train::FunctionMap ret;
 
-  // TODO Generate TrainableSeqeunce
-  // for (auto op_ind : _tdata->op_order)
-  // {
-  //   auto fn_seq = kernel_gen->generate(op_ind);
-  //   ret.emplace_back(op_ind, std::move(fn_seq));
-  // }
+  for (auto op_ind : _tdata->op_order)
```
Comment (glistening) [inline runtime/onert/backend/train/BackendContext.cc:77]:
```suggestion
  for (const auto &op_ind : _tdata->op_order)
```

#11011

**Comment by glistening** [comment]
@jyoungyun Sorry for bothering you. As you already approved, `auto &&` is applied. It is your preference. I am okay for only fixing conflicting file.

**Comment by jyoungyun** [comment]
> @jyoungyun Sorry for bothering you. As you already approved, auto && is applied. It is your preference. I am okay for only fixing conflicting file.

This case does not use auto && here.
```
/home/jyoung/git/npu/ONE/runtime/onert/backend/train/BackendContext.cc:77:38: error: cannot bind rvalue reference of type 'const onert::util::Index<unsigned int, onert::ir::OperationIndexTag>&&' to lvalue of type 'onert::util::Index<unsigned int, onert::ir::OperationIndexTag>'
   77 |   for (const auto &&op_ind : _tdata->op_order)
      |                                      ^~~~~~~~
```

**Review by glistening — APPROVED** [review]
LGTM

**Review by ragmani — APPROVED** [review]
LGTM

**Pull request merged**